src/pages/overview/pdf-electronic-seal-api/gettingstarted.md

```diff
@@ -372,7 +372,7 @@ Jumpstart your development by bookmarking or downloading the following key resou
 
 The samples project requires the following:
 
--   NET: version 6.0 or above
+-   NET: version 8.0 or above
 -   A build Tool: Either Visual Studio or .NET Core CLI.
 
 #### Authentication
```

Comment:
Line 382 ".NET samples" instead of java-samples

Reply:
Fixed